### PR TITLE
ci: render contributor list in release notes

### PR DESCRIPTION
**Pull request by Telomelonia**
continue [#592](https://github.com/apify/crawlee-python/pull/592)
solves [#468](https://github.com/apify/crawlee-python/issues/468)
- [x] test in local
- [ ] Test in actions

### CODE REVIEW

File: git-cliff-release/action.yaml [hunk 1]
```diff
@@ -86,9 +103,14 @@ runs:
...
         GITHUB_REPO: ${{ github.repository }}
         GH_TOKEN: ${{ github.token }}
+        GIT_CLIFF_TEMPLATE: ${{ env.TEMPLATE_CONTENT }}
       run: |
         echo 'release_notes<<EOF' >> $GITHUB_OUTPUT
-        git-cliff --tag "${{ steps.version_number.outputs.tag_name }}" --unreleased --strip all | tee -a $GITHUB_OUTPUT
+        if [[ -n "$GIT_CLIFF_TEMPLATE" ]]; then
+          git-cliff --tag "${{ steps.version_number.outputs.tag_name }}" --unreleased --strip all | tee -a $GITHUB_OUTPUT
+        else
+          git-cliff --tag "${{ steps.version_number.outputs.tag_name }}" --unreleased --strip all | tee -a $GITHUB_OUTPUT
```
Comment (Telomelonia) [inline git-cliff-release/action.yaml:112]: I can't think of a way to handle errors or give warning in this if..else block

**Comment by Telomelonia** [comment]
@janbuchar 
I moved the PR to here to change the template for release_notes

**Comment by janbuchar** [comment]
Hi @Telomelonia! I already addressed the issue in #135, since I was already editing the files. Your research helped me a lot though! Would you like to give me some feedback on the PR?

**Comment by Telomelonia** [comment]
> Hi @Telomelonia! I already addressed the issue in #135, since I was already editing the files. Your research helped me a lot though! Would you like to give me some feedback on the PR?

Nice, let me close this one, I think that's better approach to add if..else.

**Pull request closed without merging**